readme.md

```diff
@@ -232,6 +232,55 @@ Date: Thu, 12 May 2016 23:08:42 GMT
 Location: https://localhost:8080/9
 ```
 
+## PUT
+
+Create or update a course object.
+
+#### PUT by CRN
+
+##### If course hasn't existed:
+
+```
+> PUT /api/v0/slash_courses/17915 HTTP/1.1
+Content-Type: application/json
+
+{
+  "crn": 17915,
+  "courseNum": "CS534",
+  "courseName": "CS 534. MACHINE LEARNING",
+  "slash": 1,
+  "instructor": {
+    "lastName": "Fern",
+    "firstName": "Xiaoli"
+  },
+  "day": "TR"
+}
+
+HTTP/1.1 201 Created
+Content-Length: 0
+Date: Tue, 17 May 2016 17:29:41 GMT
+Location: https://localhost:8080/222
+```
+
+##### If course has existed:
+
+```
+> PUT /api/v0/slash_courses/17915 HTTP/1.1
+Content-Type: application/json
+
+{
+    "slash": 0,
+    "term": "F16",
+    "time": "1200-1320",
+    "location": "LINC 307",
+    "type": "Lecture"
+}
```

Comment:
Yes, we can change the instructor either by assigning another instructorId or creating a new instructor object and retrieve its instructorId automatically.